concord-server/server/src/main/java/com/rabidgremlin/concord/dao/UploadDao.java

```diff
@@ -0,0 +1,22 @@
+package com.rabidgremlin.concord.dao;
+
+import com.rabidgremlin.concord.api.UnlabelledPhrase;
+import org.jdbi.v3.sqlobject.CreateSqlObject;
+import org.jdbi.v3.sqlobject.transaction.Transaction;
+
+public interface UploadDao {
+
+    @CreateSqlObject
+    VotesDao votesDao();
+
+    @CreateSqlObject
+    PhrasesDao phrasesDao();
+
+    @Transaction
+    default void deleteExistingVotesAndUploadPhrase(String phraseId, UnlabelledPhrase unlabelledPhrase) {
+        votesDao().deleteAllVotesForPhrase(phraseId);
+        phrasesDao().upsert(phraseId, unlabelledPhrase.getText(), false);
+    }
```

Comment:
I think we also want to upsert phrase possible labels (if they exist) into votes here.